src/SubStatus.h

```diff
@@ -190,6 +241,7 @@ template <> struct std::hash<SubStatus> {
         if (auto *ba = s.byteArray(); ba) return HashHasher{}(*ba);
         else if (auto *dsp = s.dsproof(); dsp) return DspHash::Hasher{}(dsp->hash);
         else if (auto *bh = s.blockHeight(); bh && *bh) return Util::hashForStd(**bh);
+        else if (auto *hs = s.hashSet(); hs && *hs) return HashHasher{}(HashSetHash(hs->value()));
```

Comment:
Also here. This is why we need the hash computation of the set of hashes. From comment above:
```
// Specialization of std::hash so we can use SubStatus with std::unordered_map, std::unordered_set, etc
```